### ISSUE TITLE
[feature] Add overlay tracking of the HMD

### ISSUE THREAD

**Issue by tingvarsson** [relamptk/vr-streaming-overlay#16]
Basically what the titles says. Make the overlay track the HMD instead of the controller. Basically make it a static position relative the eyes.

**Comment by beniwtv** [comment]
Yes, that it quite easy to achieve, actually. It might not be everyone's cup of tea, due to nausea though.

**Comment by tingvarsson** [comment]
Aha, hadn't occurred to me that it could be an issue!
I thought of it partly because I had issues getting the hand based tracking to work pleasantly, or perhaps rather understanding how it is supposed to work. More specifically how the dim/undim works, it would not show at all most of the time when I tried it out. And possibly only show itself when pointing in weird angles.

**Comment by beniwtv** [comment]
The hand tracking will get an update soon for the show/hide behavior, how it is implemented right now is not intuitive. Dim/undim works by just looking at it for the x amount of seconds you specify
